acceptance/suites/puppet3_tests/020_backwards_compat/test.rb

```diff
@@ -0,0 +1,118 @@
+# require 'test/unit/assertions'
+# require 'puppet/acceptance/module_utils'
+
+test_name "3.x agent running against 4.x master"
+
+studio = "/tmp/simmons-studio-#{Process.pid}"
+
+teardown do
+  on(agents, "rm -rf #{studio}")
+end
+
+step "Install modules" do
+  simmons_version = '0.2.0'
+  on(master, puppet("module install nwolfe-simmons --version #{simmons_version}"))
+  on(master, puppet("module list"))
+end
+
+step "Configure file serving" do
+  fileserverconf = on(master, puppet("config print fileserverconfig")).stdout.chomp
+  create_remote_file(master, fileserverconf, <<FILESERVERCONF)
+[simmons_custom_mount_point]
+path /etc/puppetlabs/code/environments/production/modules/simmons/mount-point-files
+allow *
+FILESERVERCONF
+  on(master, "chmod 644 #{fileserverconf}")
+end
+
+step "Disable auth.conf" do
```

Comment:
I'd generally agree with @jeffmccune here that executing with the default set of auth.conf rules in place would be ideal.  I'm also curious what specifically in the auth.conf has to be disabled in order for these tests to work.